forms/AddressFieldset/index.js

```diff
@@ -117,23 +118,23 @@ export default React.createClass({
     ])
 
     return (
-      <div className={ classes }>
-        <div className={ `hui-AddressFieldset__wrap hui-AddressFieldset__wrap--internal-${props.internalSpacing}` }>
-          <div className="hui-AddressFieldset__header">
-            { props.header }
+      <div className={classes}>
+        <div className={`hui-AddressFieldset__wrap hui-AddressFieldset__wrap--internal-${props.internalSpacing}`}>
+          <div className='hui-AddressFieldset__header'>
```

Comment:
Take that up with the author of StandardJS :P

Cheers,
Justin

> On 5 Sep. 2016, at 12:19 pm, benoneal <notifications@github.com> wrote:
> 
> In forms/AddressFieldset/index.js:
> 
> > @@ -117,23 +118,23 @@ export default React.createClass({
> >      ])
> >  
> >      return (
> > -      <div className={ classes }>
> > -        <div className={ `hui-AddressFieldset__wrap hui-AddressFieldset__wrap--internal-${props.internalSpacing}` }>
> > -          <div className="hui-AddressFieldset__header">
> > -            { props.header }
> > +      <div className={classes}>
> > +        <div className={`hui-AddressFieldset__wrap hui-AddressFieldset__wrap--internal-${props.internalSpacing}`}>
> > +          <div className='hui-AddressFieldset__header'>
> JSX string properties use double-quotes. JSX === looks like HTML, only curly braces break out into regular js.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>